src/litserve/python_client.py

```diff
@@ -13,5 +14,6 @@
 # limitations under the License.
 import requests
 
-response = requests.post("http://127.0.0.1:{{PORT}}/predict", json={"input": 4.0})
-print(f"Status: {response.status_code}\nResponse:\n {response.text}")
+response = requests.post("http://127.0.0.1:{PORT}/predict", json={{"input": 4.0}})
+print(f"Status: {{response.status_code}}\\nResponse:\\n {{response.text}}")
+"""
```

Comment:
Just FIY, you can also use [Template](https://docs.python.org/3/library/string.html#template-strings). So you don't have to add double curly braces.
Here it's not needed, but if you will ever create a large template, it might come in handy.

It will be something like this:
```python
from string import Template

client_template = Template("""
# Copyright The Lightning AI team.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# limitations under the License.
import requests

response = requests.post("http://127.0.0.1:${PORT}/predict", json={"input": 4.0})
print(f"Status: {response.status_code}\\nResponse:\\n {response.text}")
""")
```

and to use it
```python
client_template.substitute(PORT=port)
```

Reply:
This is great! Didn't know about you. Thanks for letting me know.

Reply:
I guess I can also use it for Dockerfile template